### ISSUE TITLE
Improve support for Andes

### ISSUE THREAD

**Issue by forsyth2** [E3SM-Project/zppy#450]
Improve support for Andes

Automatically determine the `account` at https://github.com/E3SM-Project/zppy/blob/main/zppy/__main__.py#L143:
```
    if config["default"]["account"] == "":
        if default_account:
            config["default"]["account"] = default_account
        elif config["default"]["machine"] in ["compy", "pm-cpu", "pm-gpu", "chrysalis"]:
            config["default"]["account"] = "e3sm"
        elif config["default"]["machine"] == "anvil":
            config["default"]["account"] = "condo"
        else:
            raise ValueError(f"Invalid machine {config['default']['machine']}")
```

Add the following to https://github.com/E3SM-Project/zppy/blob/main/zppy/templates/slurm_header.sh (from @whannah1):
```
{% elif machine == 'andes' %}

# Running on andes

#SBATCH  --job-name={{ prefix }}
#SBATCH  --account={{ account }}
#SBATCH  --nodes={{ nodes }}
#SBATCH  --output={{ scriptDir }}/{{ prefix }}.o%j
#SBATCH  --exclusive
#SBATCH  --time={{ walltime }}
#SBATCH  --partition={{ partition }}
```
[Lines already existing in the file won't need to be duplicated]

**Comment by forsyth2** [comment]
Note that https://github.com/E3SM-Project/mache/blob/main/mache/machines/andes.cfg has no `web_portal` section.

**Comment by xylar** [comment]
> Note that https://github.com/E3SM-Project/mache/blob/main/mache/machines/andes.cfg has no web_portal section

Some machines don't have a web portal, so there's no `web_portal` section.  I thought that was the case for Andes but if there is a web portal that I don't know about, please let me know so I can add it.

**Comment by forsyth2** [comment]
@xylar I'm not sure if there is a web portal, but I wanted to note the issue here. @whannah1 found that `zppy` will fail because of the missing web portal. The best solution might be to have some sort of `skip-web` option/parameter.

**Comment by xylar** [comment]
Yes, something like that sounds necessary in the case of Andes.  I looked around and Im' pretty sure there's not a web portal visible to the public at OLCF.
